### DISCUSSION TITLE
Video not playing at safari macOS.

### DISCUSSION THREAD

**Discussion by ktjd123** [community/community#39623] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi,

It seems like video (tested extension .mov) is available to play in macOS Safari.

Chrome works fine.

Thank you for your great works. :)

**Comment by ossanna16** [comment]
Hi @ktjd123! Can you please specify which video you are referring to?

**Comment by ktjd123** [comment, marked as answer]
Hi, @ossanna16. Thank you for reply.

It seems like problem is solved now!

**Reply by ossanna16** [reply]
Perfect :)
